### DISCUSSION TITLE
excalidraw implementation discussion

### DISCUSSION THREAD

**Discussion by sohailamjad256** [excalidraw/excalidraw#4612] [Q&A]
Hi..can we give one button  in toolbar to maximize the screen and when we click back on same button then it comes to original form

**Comment by dwelle** [comment]
You can use one of the custom UI render props (`renderTopRightUI`, `renderFooter`) to render that button. For the implementation of the fullscreen button, for now I think you'll have to copy what we're doing:

https://github.com/excalidraw/excalidraw/blob/d9cc7d103332f77e2cf380dd4c868e8b4cd2916a/src/actions/actionMenu.tsx#L57-L62

That said, we should add support for host apps to execute editor actions. Can you create an issue for this?

**Reply by sohailamjad256** [reply]
can you please  send the clip..if you have ?

**Reply by dwelle** [reply]
what do you mean?
